### PR TITLE
[DOCFIX] Add new sections to CLI docs

### PR DESCRIPTION
**Pull request by ns1123**
Add sections to CLI docs

### CODE REVIEW

**Comment by ns1123** [comment]
@LuQQiu PTAL :)

File: docs/en/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -187,6 +203,11 @@ The usage of this command includes:
 -Dalluxio.underfs.s3.endpoint=<endpoint_url> -Dalluxio.underfs.s3.disable.dns.buckets=true
 ```
 
+### readJournal
```
Comment (LuQQiu) [inline docs/en/basic/Command-Line-Interface.md]: `readJournal` is a developer facing command that will be better to put in the `contributor-tool.md` and I updated this docs in https://github.com/Alluxio/alluxio/pull/9249/files according to @ggezer 's newest journalTool changes

File: docs/en/basic/Command-Line-Interface.md [hunk 2]
```diff
@@ -155,6 +166,11 @@ And the following command returns the log level of the class `alluxio.heartbeat.
 alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
 ```
 
+### runClass
```
Comment (LuQQiu) [inline docs/en/basic/Command-Line-Interface.md]: This is developer facing command that will be better to put in `Contributor-tools.md`

File: docs/en/basic/Command-Line-Interface.md [hunk 3]
```diff
@@ -26,6 +26,11 @@ Usage: alluxio [COMMAND]
 This section lists usages and examples of general Alluxio operations with the exception of file
 system commands which are covered in the [Admin CLI doc]({{ '/en/operation/Admin-CLI.html' | relativize_url }}).
 
+### extensions
+
+The `extensions` command is meant to for managing any extensions to Alluxio. For additional information, refer
```
Comment (LuQQiu) [inline docs/en/basic/Command-Line-Interface.md]: `any` -> `UFS`. Now `extensions` only used to install UFS extensions.

File: docs/en/basic/Command-Line-Interface.md (hunk 2 above)
Comment (ns1123) [reply docs/en/basic/Command-Line-Interface.md]: done

File: docs/en/basic/Command-Line-Interface.md (hunk 3 above)
Comment (ns1123) [reply docs/en/basic/Command-Line-Interface.md]: done

File: docs/en/basic/Command-Line-Interface.md (hunk 1 above)
Comment (ns1123) [reply docs/en/basic/Command-Line-Interface.md]: removed

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**